src/dotnet/Program.cs

```diff
@@ -78,14 +76,15 @@ public static int Main(string[] args)
             }
         }
 
-        internal static int ProcessArgs(string[] args, ITelemetry telemetryClient)
+        internal static int ProcessArgs(string[] args, ITelemetry telemetryClient = null)
```

Comment:
This processargs is used for unit testing in one of the tests, and it passes a mock telemetry object.